packages/lumx-react/src/components/flag/Flag.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { Theme, ColorPalette, Icon } from '@lumx/react';
+import { GenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+import { ColorVariant } from '..';
+
+export interface FlagProps extends GenericProps {
+    /** Theme adapting the component to light or dark background. */
+    theme?: Theme;
+    /** The color of the component */
+    color?: ColorPalette;
+    /** icon */
+    icon?: string;
+    /** label of the flag */
+    label: string;
```

Comment:
Could you please standardize the jsdoc? This means dropping the `The` formalism, put an uppercase at the beginning of the sentence and a dot at the end.
This is less important inside the product codebase because the product jsdoc is poorly used. However, the jsdoc of the DS is used inside the public documentation (in the table props).